celsos_bookSearch/celsos_bookSearch/Services/GetService.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  GetService.swift
+//  celsos_bookSearch
+//
+//  Created by Celso Junio Simões de Oliveira Santos on 21/05/21.
+//
+
+import Foundation
+import PromiseKit
+
+//class GetService
+// TO DO
+// Formato tirado de um POST. GET a construir
+class GetService {
+
+    static var baseURL = ""
+
+    func postLogin(_ emailText: String, _ passwordText: String) {
```

Comment:
When you create a method like this suppressing the argument name makes it hard to get if we are sending the right parameter at the right position because the call of it would be something like:
`getService.postLogin(firstParameter, secondParameter)`
We have the possibility to let it explicit and make the code more readable:
`func postLogin(email: String, password: String) {`
That way at the usage point it would be clear which parameter should receive each info
`getService.postLogin(email: firstParameter, password: secondParameter)`